plugins/acss-kit/assets/foundation/SOURCE.md

```
@@ -0,0 +1,171 @@
+For each file, run:
+
+```bash
+# Example for _globals.scss — repeat for each file in the tree
+gh api "repos/shawn-sandy/acss/contents/packages/fpkit/src/sass/_globals.scss?ref=<new-sha>" \
+  --jq '.content' | base64 -d \
+  > plugins/acss-kit/assets/foundation/sass/_globals.scss
+```
+
+Files to refresh (15 total):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)